release-notes/2023-03/README.md

```diff
@@ -0,0 +1,71 @@
+# SystemLink Enterprise 2023-03 Release Notes
+
+The 2023-03 bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Support for Kubernetes 1.23
+
+- Support for both PostgreSQL 13 and 14.
+
+- Product, results, and steps tables can be queried using the table query builder.
+
+- OpenTelemetry metrics available for the following services
+    - testmonitor
+    - dataframeservice
+    - Kafka
+
+- The endpoint to view claims for the current logged in user `/oidc/userinfo` has been removed and replaced with `oidc/user-info`.
+
+- Kafka UI is now available
+
+- Increase the default memory provided to Kafka connect from XXX to YYY.
+
+## Helm Chart Breaking Changes
+
+- systemlink TODO Version
+    - Enabling Kafka OpenTelemetry metrics
+        - Set `global.telemetry.enabled` flag to `true`.
+        - You are no longer required to also set options in `kafka.metricsConfigs`, `kafka.jmxOptions`, and `kafka.kafkaExporter` configurations
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-03/getting-started/templates/systemlink-values.yaml#L56)
+
+    - Enabling user claims to be returned in a web browser
+        - Set `webserver.oidc.enableUserInfo` to `true`
+        - You should remove `offline_access` from `webserver.oidc.scope`.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-03/getting-started/templates/systemlink-values.yaml#L110)
+
+    - Enabling Kafka UI
+        - To enable the Kafka UI you must override the image pull repository to the repository you are installing SystemLink Enterprise from.
```

Comment:
They will also need to override the image pull secrets value. I've updated the helm breaking changes spreadsheet with details.

Reply:
@cameronwaterman this should be resolved, but please review the comments on line 33 of the release notes